main/gy521_imu.h

```diff
@@ -70,14 +70,25 @@ void setupIMU(){
     Serial1.println("IMU setup successful.");
   else{
     Serial1.println("Setup failed, did not read acceleration values from IMU.");
-    Serial1.print("X Acceleration: ");
-    Serial1.println(sample.accel.x);
-    delay(1000);
-    Serial1.print("Y Acceleration: ");
-    Serial1.println(sample.accel.y);
-    delay(1000);
-    Serial1.print("Z Acceleration: ");
-    Serial1.println(sample.accel.z);
-    delay(1000);
-  }
+    } 
```

Comment:
Could you shift this bracket to the left by one tab so it's lined up with the start of the word `else`?

Reply:
Done!